texar/torch/modules/connectors/connectors.py

```diff
@@ -16,8 +16,8 @@
 """
 
 from typing import Any, Callable, Dict, List, Optional, Tuple, Union
-
```

Comment:
As a reference, the preferred way to format imports is:

- Split imports into three parts: builtin packages, third-party packages, our own packages. Parts are listed in this order and different parts are separated by one empty line.
- Within each part, sort imports in lexicographical order by their package names. For the same library `xxx`, `import xxx` should come before `from xxx import yyy`.
- My personal preference is to also sort imported identifiers in `from ...` in case-sensitive lexicographical order. But you don't have to do this.

Before this change, the imports in this file were correct.

Reply:
Great explanations, thanks!